scripts/mkcloud

```diff
@@ -950,6 +950,8 @@ function cloudupgrade()
 {
     if iscloudver 5plus ; then
         crowbarupgrade_5plus
+        onadmin crowbar_shutdown_services
+        onadmin crowbar_openstack_backup
```

Comment:
Hm, that depends on what you mean with "crowbarupgrade". If that refers to "Upgrading the Admin Node" then these steps are not part of the crowbarupgrade. They perform actions on the client nodes (e.g. stopping openstack and creating a backup of the postgresql database). In that case I'd argue they are more related to the nodeupgrade step and should stay together with that.

But maybe I am misunderstanding what you mean by "crowbarupgrade".